NOTES.ANDROID

```diff
@@ -0,0 +1,66 @@
+
+ NOTES FOR ANDROID PLATFORMS
+ ===========================
+
+ Requirement details
+ -------------------
+
+ Beside basic tools like perl and make you'll need to download the Android
+ NDK. It's available for Linux, Mac OS X and Windows, but only Linux
+ version was actually tested. There is no reason to believe that Mac OS X
+ wouldn't work. And as for Windows, it's unclear which "shell" would be
+ suitable, MSYS2 might have best chances. NDK version should play lesser
+ role, the goal is to support a range of most recent versions.
+
+ Configuration
+ -------------
+
+ Android is naturally cross-compiled target and you can't use ./config.
+ You have to use ./Configure and name your target explicitly; there are
+ android-arm, android-arm64, android-mips, android-mip64, android-x86
+ and android-x86_64. Do not pass --cross-compile-prefix (as you might
+ be tempted), as it will be "calculated" automatically based on chosen
+ platform. Though you still need to know the prefix to extend your PATH,
+ in order to invoke $(CROSS_COMPILE)gcc and company. (Configure will fail
+ and give you a hint if you get it wrong.) Apart from PATH adjustment
+ you need to set ANDROID_NDK environment to point at NDK directory
+ as /some/where/android-ndk-<ver>. NDK customarily supports multiple
+ Android API levels, e.g. android-14, android-21, etc. By default latest 
+ one available is chosen. If you need to target older platform, pass
+ additional -D__ANDROID_API__=N to Configure. For example, to compile for
```

Comment:
N is not clear in this context. What about "...=NUMBER, where NUMBER is numeric value of target platform version ..."

Reply:
Well, it's arguably more or less obvious as numbers are mentioned above and below as examples, but sure...